### DISCUSSION TITLE
1 click button to refilter the bank with 'inventory setups'

### DISCUSSION THREAD

**Discussion by Mocklies** [runelite/runelite#15794] [Ideas]
With inventory setups, going between tabs will stop the filtering and you have to go out of the setup and then back in. (not a big deal but i find it annoying haha)

Suggestion: add a button that does like a refresh of the filtering in the bank to save 1 click.

**Comment by Nightfirecat** [comment]
Inventory Setups is a hub plugin, we do not maintain it. You can request this change from them: https://github.com/dillydill123/inventory-setups
